packages/utils/src/fetch.ts

```diff
@@ -32,7 +32,7 @@ async function unwrappedRdfFetch(
     requestInit.headers = { Accept: 'text/turtle', ...requestInit.headers };
   }
   const response = await whatwgFetch(iri, requestInit);
-  const rdfResponse = { ...response } as RdfResponse;
+  const rdfResponse = response.clone() as RdfResponse;
```

Comment:
Copying a response with object destructuring does not work, the `.clone` method has to be used. However this breaks the tests as the mocks do not implement it, @elf-pavlik can you take a look at fixing the mocks?